bedrock/firefox/views.py

```diff
@@ -552,6 +552,10 @@ def new(request):
                 template = 'firefox/new/batm/nimble.html'
             elif experience == 'batmresist':
                 template = 'firefox/new/batm/resist.html'
+            elif experience == 'batma':
```

Comment:
It would be nice to [add some tests](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/tests/test_views.py#L494-L504) for these new variations?